app/views/pages/_contributor_blurb.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="gridManyItem center">
+  <%= react_component('Modal', props: {
+    element: { component: 'Avatar', props: { src: ProfilePicture.normalize_url(blurb["image"]), name: blurb['name'], alt: "Contributor: #{blurb['name']}"} },
```

Comment:
Ooh! This is a good workaround 🙌 

Since we support i18n, we should have translations for that contributor text, maybe it could be, "Contributor biography for %{name}".

You can go ahead as use Google translate for now and we'll have the appropriate people review the translations later :P